src/lib/data/my-account-model.ts

```diff
@@ -0,0 +1,8 @@
+import { UserPii } from "./db-models";
+
+export type MyAccount = {
```

Comment:
How come this isn't in `user-models`? Isn't it a type for the getMyAccount user action?

Reply:
I think I'm still familiarising myself with the organisation haha!

Done 32e0d54382111d2fa83619d9f5737419e80f94e2

Reply:
Ha ha. Sorry for the ever changing organisation.